pkg/api/org.go

```diff
@@ -49,14 +50,13 @@ func (hs *HTTPServer) GetOrgByName(c *models.ReqContext) response.Response {
 	return response.JSON(200, &result)
 }
 
-func getOrgHelper(orgID int64) response.Response {
+func getOrgHelper(ctx context.Context, orgID int64) response.Response {
 	query := models.GetOrgByIdQuery{Id: orgID}
 
-	if err := sqlstore.GetOrgById(&query); err != nil {
+	if err := sqlstore.GetOrgById(ctx, &query); err != nil {
 		if errors.Is(err, models.ErrOrgNotFound) {
-			return response.Error(404, "Organization not found", err)
+			return response.Error(500, "Failed to get organization", err)
```

Comment:
I might misunderstand this, but why `500` instead of `404`? Would an invalid `orgId` result in `500` error?

Reply:
this is some rebase mishap, I wll change it back